intel-retail/automated-vending
# feat(ms-inventory): Migrate ms-inventory to EdgeX v3.0

## PR Checklist



## What are you changing?


## Issue this PR will close

close: #**129**

closes #129

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable


- Comment out lines, 8-31, lines 58 - 238 and line 240 in `docker-compose.ac.yml` (so that only ms-inventory service and inventory volume are not commented out)
- Run `make docker` in ms-inventory dir
- Run `make run` in automated-vending dir
- Run `make run-portainer` and check ms-inventory logs

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-vending )
